catalog/catalog.yaml

```diff
@@ -155,7 +155,6 @@ components:
           description: A list of URLs that represent the audience for the token, which is needed for some client credential grant flows.
           x-go-type-skip-optional-pointer: true
         tokenMetadataFields:
-          description: Fields to be used to extract token metadata from the token response.
```

Comment:
is there a reason why you deleted these descriptions? If they are inaccurate, can you please update instead of delete?